packages/seed-data-workflow/package.json

```diff
@@ -19,8 +19,8 @@
     "eslint-plugin-prettier": "^5.1.3"
   },
   "dependencies": {
-    "@reapit/connect-session": "^6.1.1",
-    "@reapit/connect-session-server": "^1.1.0",
+    "@reapit/connect-session": "^6.1.3",
+    "@reapit/elements": "^4.3.2",
```

Comment:
not sure what's going on here but checking the imports within src, only connect-session-server is imported. Elements is also not imported. Think this might've been a mistake